Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/D7YIGPositionCalibration.py

```diff
@@ -186,9 +186,12 @@ def _get_scan_data(self, ws_name):
             # normalize to monitor1 as monitor2 is sometimes empty:
             monitor1_counts = entry.readY(self._D7NumberPixels-2)[0]
             if monitor1_counts != 0:
+                CreateSingleValuedWorkspace(DataValue=monitor1_counts, ErrorValue=np.sqrt(monitor1_counts),
+                                            OutputWorkspace='_monitor')
```

Comment:
Even though this workspace is really temporary, you should put some user input string as prefix or suffix just in case of several parallel reductions. Moreover, if you need it hidden, it's double underscore that you need.